source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -0,0 +1,295 @@
+and [Alex Speller](https://github.com/alexspeller) have contributed a series of
+patches to address some long-standing issues and edge cases in the routing
+layer, particularly in the area of query params.
+
+Second, thanks to [Robert Jackson](https://github.com/rwjblue), we now use
+[WeakMap](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WeakMap)
+to [store object metadata](https://github.com/emberjs/ember.js/pull/13783) on
+browsers that supports them. A plesant side-effects of this internal change is
+that Ember no longer add an `__ember_meta__` property to every object.
+
+Finally, it has come to our attention that it is quite common for developers to
+accidentally mutate the arguments passed into a helper. This is problematic as
+these objects are often shared or reused, causing subtle issues that are very
+difficult to debug.
+
+To help track down these kind of issues, these objects are now [frozen](https://github.com/emberjs/ember.js/pull/14244)
+in the debug build before they are passed into helpers. As a result, you will
+get an error if you attempt to modify these objects directly. It is recommended
+that you clone these objects (e.g. using `Object.assign` and `Array#slice`)
+before making any modifications to them.
+
+(It has been brought to our attention that the last change might have an
+[unintended consequence](https://github.com/emberjs/ember.js/pull/14649).
+A fix is being worked on by [Matthew Beale](https://github.com/mixonic). Please
+refer to the linked issue for a detailed description and suggested workaround.)
+
+For more details on the changes in Ember.js 2.10, please review the
+[Ember.js 2.10.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.10.0).
+
+### Upcoming changes in Ember.js 2.11
+
+Currently, no new features are planned for 2.11. However, there are a few
+notable changes:
+
+- Similar to the change to helper arguments, concatenated properties (such as
+  `classNames` and `classNameBindings`) are now [frozen](https://github.com/emberjs/ember.js/pull/14389)
+  in debug builds to help track down unintended mutations.
+
+- The legacy `render` helper (i.e. `{{render 'foo'}}`) has been [deprecated](https://github.com/emberjs/ember.js/pull/14441).
+
+- The private `Component#renderToElement` API has also been [deprecated](https://github.com/emberjs/ember.js/pull/14482).
+
+For more details on the upcoming changes in Ember.js 2.11, please review the
+[Ember.js 2.11.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.11.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the offical data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.10
+
+Ember Data 2.10 represents the work of more than 10 direct contributors with over
+20 commits.
+
+There have been no user facing changes in Ember Data 2.10. Under the
+hood, [@runspired](https://github.com/runspired) has instrumented
```

Comment:
Should we normalize these mentions to use real names (see the Ember section above)? @runspired do you mind?